src/main/java/harmonised/pmmo/features/mobscaling/MobAttributeHandler.java

```diff
@@ -57,11 +61,9 @@ public static void onBossAdd(EntityJoinLevelEvent event) {
 	public static void onMobSpawn(FinalizeSpawn event) {
 	    if (!Config.MOB_SCALING_ENABLED.get())
 	        return;
-		if (event.getEntity().getType().is(Reference.MOB_TAG)) {
-			handle(event.getEntity(), event.getLevel().getLevel()
-					, new Vec3(event.getX(), event.getY(), event.getZ())
-					, event.getLevel().getDifficulty().getId());
-		}
+		handle(event.getEntity(), event.getLevel().getLevel()
```

Comment:
Restore the use of the mob tag.  This is there to prevent things like vanilla animals from scaling.  If your modpack wants animals to scale, you need to add them to this tag.